inbox.scss

```diff
@@ -13,11 +13,8 @@ v-grid td.v-grid-cell * {
     td.v-grid-cell {
         border-left: none !important;
         border-right: none !important;
-        padding: 12px !important;
-    }
-
-    .v-grid-row-stripe:not(.v-grid-row-selected) > td {
-        background: #fff;
+        min-height: 62px;
```

Comment:
Forgot to add this as a comment in the stylesheet:
Grid calculates the row heights by adding a temporary `<td></td>` with an ìnnerText` and uses the height from that cell to calculate rest of the heights. It doesn't take custom renderers into account so in this case we need to set the cell `min-height` for everything to work out fine and dandy.